src/debug/daccess/dacdbiimpl.cpp

```diff
@@ -839,7 +839,14 @@ SIZE_T DacDbiInterfaceImpl::GetArgCount(MethodDesc * pMD)
 // Allocator to pass to DebugInfoStores, allocating forDBI
 BYTE* InfoStoreForDbiNew(void * pData, size_t cBytes)
 {
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wnull-dereference"
```

Comment:
I am finishing a proper fix for this problem (all the cases of null-dereference warnings) and I'll send a PR for it today, so please wait for it and remove the warning disabling.